Blockzilla/AppDelegate.swift

```diff
@@ -18,6 +19,32 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             BuddyBuildSDK.setup()
         #endif
 
+        // Set up Telemetry
+        let telemetryConfig = Telemetry.default.configuration
+        telemetryConfig.appName = "Focus"
+        telemetryConfig.dataDirectory = .documentDirectory
+        telemetryConfig.userDefaultsSuiteName = AppInfo.sharedContainerIdentifier
+        
+        telemetryConfig.measureUserDefaultsSetting(forKey: "prefKeyEngine", withDefaultValue: SearchEngineManager(prefs: UserDefaults.standard).engines.first?.name)
+        telemetryConfig.measureUserDefaultsSetting(forKey: SettingsToggle.blockAds.rawValue, withDefaultValue: Settings.getToggle(.blockAds))
```

Comment:
I just found a really cool way to handle this generically in the library. I just added this overloaded method definition:

```
public func measureUserDefaultsSetting<T : RawRepresentable>(forKey key: T, withDefaultValue defaultValue: Any?) where T.RawValue == String
```